### PR TITLE
Connect Redux to project

### PR DESCRIPTION
**Pull request by jodavid444**
# In This Pull Request:

- Connect redux to react components
- Combine both reducer and use dispatch.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -6,22 +6,22 @@ const books = [];
 const BookReducer = (state = books, action) => {
   switch (action.type) {
```
Comment (Owusu-Desmond) [inline src/redux/books/books.js]: - [ ] Please from the project requirement, You should define `initialState` that contains some books. like this :point_down: 
  ```javascript
  const initialState = {
    books: [
      {
        item_id: "item1",
        title: "The Great Gatsby",
        author: "John Smith",
        category: "Fiction",
      },
      {
        item_id: "item2",
        title: "Anna Karenina",
        author: "Leo Tolstoy",
        category: "Fiction",
      },
      {
        item_id: "item3",
        title: "The Selfish Gene",
        author: "Richard Dawkins",
        category: "Nonfiction",
      },
    ],
  }
  ```

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @jodavid444 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @jodavid444, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jodavid444** [comment]
Thanks for the review @topeogunleye

**Comment by topeogunleye** [comment]
> Thanks for the review @topeogunleye

@jodavid444 you're welcome

**Pull request merged**